packages/charts/react-charting/src/components/LineChart/LineChart.types.ts

```diff
@@ -87,12 +87,6 @@ export interface ILineChartProps extends ICartesianChartProps {
    * The prop used to enable the perf optimization
    */
   enablePerfOptimization?: boolean;
-
-  /**
-   * @default default
-   * The prop used to define line chart mode(default or scatter)
-   */
-  lineMode?: 'default' | 'scatter';
```

Comment:
This cannot be removed right away. Set it as deprecated. And remove after few releases.